environment.yml

```diff
@@ -2,33 +2,33 @@
 # mamba env create -f environment.yml
```

Comment:
I listed a few issues from the environment.yml file but I may have missed some others, I think you need to check how it was merged

Reply:
Indeed, I handled the merge poorly. Rebased and corrected the mistakes. Should be better now!